### PR TITLE
Suite style fix3

### PR DESCRIPTION
**Pull request by Ayagoumi**
- Fixed issue with scrolling within the modal body
- Added a pointer cursor to all links

Screenshot 2023-02-13 at 18 17 01

### CODE REVIEW

File: src/components/modals/Modal.vue [hunk 1]
```diff
@@ -57,6 +59,10 @@ export default class Modal extends Vue {
     position: relative;
     padding: 10px 22px;
     display: flex;
+    z-index: 1102;
```
Comment (Chriserus) [inline src/components/modals/Modal.vue:62]: Is 1102 number really necessary here? Asking, as it's really specific, isn't 1000 enough?

File: src/views/wallet/Activity.vue [hunk 2]
```diff
@@ -371,7 +371,7 @@ export default class Activity extends Vue {
 
 .explorer_warning {
     position: absolute;
-    background-color: var(--bg);
+    // background-color: var(--bg);
```
Comment (Chriserus) [inline src/views/wallet/Activity.vue]: Please remove if commented out

File: src/components/modals/Modal.vue (hunk 1 above)
Comment (Ayagoumi) [reply src/components/modals/Modal.vue:62]: Yes, the default z-index of MUI AppBar is 1100.

**Review by Chriserus — APPROVED** [review]
LGTM, but I think @mo-c4t fixed cypress tests, so we should go green before merging

File: src/views/wallet/Activity.vue (hunk 2 above)
Comment (Ayagoumi) [reply src/views/wallet/Activity.vue]: Removed

**Review by aeddaqqa — APPROVED** [review]
LGTM

**Pull request merged**